Lib/multiprocessing/shared_memory.py

```diff
@@ -253,6 +254,29 @@ def unlink(self):
             if self._track:
                 resource_tracker.unregister(self._name, "shared_memory")
 
+    def rename(self, newname, flags = 0):
```

Comment:
I don't see any precedent for argument defaults in this file, so let's just go with PEP 8:
```suggestion
    def rename(self, newname, flags=0):
```